part_1/title_I/sekcja_2_PL.md

```diff
@@ -0,0 +1,27 @@
+### <a name="section_2"></a><p align="center">Artyku� 2</p>
+
+Definicje niekt�rych poj��
+
+(1) Rodzaje broni i amunicji s� okre�lone w za��czniku nr 1 do tej ustawy.
+
+(2) Na potrzeby niniejszej Ustawy rozumie si� nast�puj�co:
+
+a) przez posiadanie broni i amunicji rozumie si�
+
+1. bro� lub amunicj� w obr�bie lokalu mieszkalnego, lub handlowego _[miejsca prowadzenia dzia�alno�ci, przyp. t�umacza]_, lub wewn�trz wyznaczonego, wyra�nie odgraniczonego obszaru nieruchomo�ci za zgod� w�a�ciciela lub dzier�awcy tych lokali lub nieruchomo�ci;
```

Comment:
'bytových nebo provozních prostor' -> na stronie [czeskiego słownika do angielskiego](https://slovnik.seznam.cz/cz-en/?q=provozn%C3%AD) znalazłam takie tłumaczenie 'provozních', które wskazuje na to, że chodzi tu o lokal **użytkowy** (podobne jak w tłumaczeniu na stronie 'domovní a provozní prostory' → 'housing and utility premises'). U nas stosuje się właśnie określenie 'lokal użytkowy' - w sensie lokalu, w którym można prowadzić działalność nie tylko handlową, ale również usługową, magazynowanie itp. Lokal handlowy to 'komerční postor'.